cargo-nextest/src/dispatch.rs

```diff
@@ -2318,7 +2340,7 @@ mod tests {
         fn get_test_filter_builder(cmd: &str) -> Result<TestFilterBuilder> {
             let app = TestCli::try_parse_from(shell_words::split(cmd).expect("valid command line"))
                 .unwrap_or_else(|_| panic!("{cmd} should have successfully parsed"));
-            app.build_filter.make_test_filter_builder(vec![])
+            app.build_filter.make_test_filter_builder(vec![], todo!())
```

Comment:
Ahh this is annoying for sure. The reason package graphs are needed by the filterset logic is that they're used for package queries. But if you switch the logic to not using filtersets, you should no longer need this.